main.tex

```diff
@@ -115,11 +115,12 @@ \section{Introduction} \label{sec:intro}
 common to many astronomical tasks.
 The users of such software then also benefit from a community and ecosystem
 built around a common foundation.
-The \astropy project has grown to become this community, and the \astropypkg
-core package a feature-rich library for \python-based astronomical software.
+The \astropy project has grown to become this community for \python astronomy
+software, and the \astropypkg core package a feature-rich library for
+\python-based astronomical software.
```

Comment:
I feel the repetition (astronomy software) is not very elegant, could we rephrase? maybe "\astropypkg core package a feature-rich python library"?